### ISSUE TITLE
fman <bad-name> returns nothing

### ISSUE THREAD

**Issue by milancurcic** [urbanjost/fman#1]
If a bad name is passed to `fman`, it returns nothing (which suggests success), but it should print an error message, perhaps "No such topic" or similar.

```
$ fman bad # no such name as bad; should error out but returns nothing
$ echo $?
0 # suggests "fman bad" was successful
```

Should be straightforward to fix. I can't fix it right now, but thought to just report it.

Thank you for making fman.

**Comment by urbanjost** [comment]
Good point.  I am getting inspired to polish this up.  I had a version that added color as a test for a prototype ANSI in-line escape sequence module I was toying with around here somewhere that was promising (well, if you like color). If it is going to get some real usage I hope everyone keeps enhancements and suggestions coming.

**Comment by urbanjost** [comment]
A quick fix is in.  Not sure I like the traceback with non-zero exit in the debug mode; fine in release mode. I __really__ wish if you gave STOP a null string it would quit printing "STOP".

**Comment by urbanjost** [comment]
Why on __earth__ does STOP not allow a value and a string at the same time for that matter(just venting  a pet peeve!)?

**Comment by milancurcic** [comment]
Great, thanks!

See Section 5.8 of John Reid's [What's new in F2018](https://isotc.iso.org/livelink/livelink?func=ll&objId=19441669&objAction=Open).

You can do:

```
stop rc, quiet=.true.
```

**Comment by urbanjost** [comment]
Perfect. Missed that. Looks like I need to change the man-page! That one needed some work anyway.

**Comment by urbanjost** [comment]
fan badname
Sorry. did not find that. Perhaps you should search the TOC. try
   fan -e TOPIC
or search the entire manual:
   fan manual -i -e TOPIC

**Comment by urbanjost** [comment]
Did that a while ago. Also added ANSI in-line color control characters if your terminal emulator supports it.
